proto/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@serverless/sdk-schema",
+  "version": "0.0.1",
+  "description": "",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "rimraf -rf ./dist && rimraf -rf ./out && buf build && buf generate && tsc --emitDeclarationOnly && rollup --config && rimraf -rf ./dist/dts"
```

Comment:
If I understand correctly, the flow is as follows:
1. `buf` generates TS files into `out`
2. TS generates typings out of `out` to `dist/index.d.ts`
3. Rollup generates JS ouf of TS (in `out`) into `dist` - here question is, can we use `esbuild` instead - as we already use this transpiler in context of this repository.

Is that correct? It'll be nice to document it in some dev documentation